source/_patterns/03-templates/02-scheduled.mustache

```diff
@@ -34,7 +34,7 @@
             <div class="schedule-page__content">
                 <br>
                 <p>
-                    <span class="fc-event" style="background-color:#cde1f1;border-color:#cde1f1;color:#111;display:inline-block;"><strong>3:40pm</strong> 09672 (Adv.)</span> <br>
+                    <span class="fc-event" style="background-color:#cde1f1;border-color:#cde1f1;color:#111;display:inline-block;"><strong>3:40pm</strong> 09672 (tmp)</span> <br>
```

Comment:
Inline styles. I understand this PR does not introduce them, they were there already, so approval is not dependent on the following, but: can they be moved to the stylesheet at some point, please?